src/main/java/step2/domain/MatchesResult.java

```diff
@@ -1,29 +1,29 @@
 package step2.domain;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 import java.util.stream.Collectors;
 
+import static step2.domain.BaseScore.getBaseMap;
+
 public class MatchesResult {
-    Map<Integer, Long> results;
+    Map<ScoreType, Long> results;
```

Comment:
외부에서 참조하지 않는 것으로 보이는데

접근 제한자를 추가해주시면 어떨까요?

```suggestion
    private Map<ScoreType, Long> results;
```